client/my-sites/plugins/controller.js

```diff
@@ -74,7 +75,8 @@ function renderSinglePlugin( context, siteUrl, isWpcomPlugin ) {
 			isWpcomPlugin: isWpcomPlugin,
 			onPluginRefresh: title => titleActions.setTitle( title )
 		} ),
-		document.getElementById( 'primary' )
+		document.getElementById( 'primary' ),
+		context.store
```

Comment:
I'm pretty uneasy about passing around the whole store in `context`. This might be a valid place to put it, but it makes me nervous, like we will end up writing a bunch of tangled components.

Do you happen to know if we have had a discussion about this (or was it in one of the Framework hangouts that I missed?)